doc/api/module.md

```diff
@@ -1101,6 +1101,28 @@ added:
 `path` is the resolved path for the file for which a corresponding source map
 should be fetched.
 
+### `module.flushCompileCache([keepDeserializedCache])`
+
+<!-- YAML
+added:
+ - REPLACEME
+-->
+
+> Stability: 1.1 - Active Development
+
+* `keepDeserializedCache` {boolean} Whether the cache read from disk and already deserialized to
```

Comment:
I am not sure if this is actually useful - it seems unlikely for users to actually purge the cache while not expecting the source code to change, or they could always do some guarding to prevent purging cache for modules with unchanged source code..